imports/api/PointCalculationUtils.js

```diff
@@ -0,0 +1,118 @@
+import { Constants } from './Constants';
+import { NewGameUtils } from './NewGameUtils';
+
+export var PointCalculationUtils = {
```

Comment:
If you're going to do this, make it generic (include HK calculation too it's much more simple anyway)